### PR TITLE
Return early if tasks is empty on job_status.

### PR DESCRIPTION
**Pull request by xiao-chen**

### CODE REVIEW

File: infra/server.py [hunk 1]
```diff
@@ -221,6 +221,8 @@ def _sort_tasks_by_duration(self, tasks):
   @cherrypy.tools.no_caching()
   def job_status(self, job_id):
     tasks = self.results_store.fetch_task_rows_for_job(job_id)
+    if len(tasks) == 0:
+      return "No tasks found for specified job_id %s" % job_id
```
Comment (toddlipcon) [inline infra/server.py]: Shouldn't this still be returning a JSON struct with the expected fields, instead of a non-JSON string?

Comment (xiao-chen) [reply infra/server.py]: Good catch, I looked at the wrong example. :)

**Comment by toddlipcon** [comment]
Oh, nm, I see github has a button for that.

**Comment by xiao-chen** [comment]
Thanks Todd!

**Pull request merged**